libs/blocks/preflight/checks/seo.js

```diff
@@ -246,33 +246,20 @@ function compareResults(result, link) {
   return true;
 }
 
-export async function checkLinks({ area, urlHash, envName }) {
-  if (urlHash && linksCache.has(urlHash)) {
-    const cachedResult = linksCache.get(urlHash);
-    return {
-      ...cachedResult,
-      details: { ...cachedResult.details, linksChecked: true },
-    };
-  }
-
-  const { spidy, preflight } = await getServiceConfig(window.location.origin, envName);
-  const spidyUrl = spidy?.url || SPIDY_URL_FALLBACK;
-  const canSpidy = await spidyCheck(spidyUrl);
-  if (!canSpidy) return connectionError();
-
-  /**
-   * Find all links with an href.
-   * Filter out any local or existing preflight links.
-   * Set link to use hlx.live so the service can see them without auth
-   * */
+export async function validLinkFilter(area = document, envName = null) {
+  const { preflight } = await getServiceConfig(window.location.origin, envName);
   const knownBadUrls = preflight?.ignoreDomains
     ? preflight?.ignoreDomains.split(',').map((url) => url.trim())
     : KNOWN_BAD_URLS;
-
   const links = [...area.querySelectorAll('a')]
     .filter((link) => {
       if (
         link.href
+        && !link.href.includes('tel:')
+        && !link.href.includes('mailto:')
+        && !link.href.startsWith('#')
+        && !link.href.startsWith('https://#')
+        && !link.href.includes('bookmark://')
```

Comment:
One more thing I would do here is to add a comment for the future us to remember why we did this. 
Also, when we move away from Spidy to something that works better for these use cases, we might miss faulty URLs because they get filtered out
```suggestion
// Added extra checks because Spidy misidentifies these URL schemes as faulty. Can be removed once we stop using Spidy.
        && !link.href.includes('tel:')
        && !link.href.includes('mailto:')
        && !link.href.startsWith('#')
        && !link.href.startsWith('https://#')
        && !link.href.includes('bookmark://')
        //
```

Reply:
@robert-bogos I have made the recommended change